CMakeLists.txt

```diff
@@ -125,7 +128,7 @@ endif()
 ivw_register_modules(all_modules)            # Add inviwo modules
 add_subdirectory(src/qt)                     # Add the Qt network editor
 add_subdirectory(apps)                       # Add all the applications, uses the modules.
-ivw_add_external_projects()                  # Add external projects
+ivw_add_external_apps()                      # Add external projects
```

Comment:
this is not just for apps, way rename it? it also makes it inconsistent with the cmake flag use and the comments...

Reply:
I’ve only seen it used for apps and thought it would be more consistent with the other names (IVW_APP).